lms/djangoapps/ccx/models.py

```diff
@@ -24,6 +26,9 @@ class CustomCourseForEdX(models.Model):
     course_id = CourseKeyField(max_length=255, db_index=True)
     display_name = models.CharField(max_length=255)
     coach = models.ForeignKey(User, db_index=True)
+    # if not empty, this field contains a compressed json serialized list of
+    # the master course modules
+    structure_json = CompressedTextField(verbose_name='Structure JSON', blank=True, null=True)
```

Comment:
From my reading of the code, this field won't actually just be empty - it might be `'null'` *or* an empty string, correct?